Adding Bestiary Images

pcoffey42 opened the pull request:
Adding images pulled from the book, and including monsters from T-Z.

pcoffey42 committed:
Uploading AotA Images

pcoffey42 committed:
Merge branch 'master' of https://github.com/rubzanatta/arcana5etools

pcoffey42 committed:
Including Monsters T-Z

luathebee commented on ArcanaOfTheAncientsBestiary.json:
```diff
@@ -8532,6 +8532,1595 @@
+			"passive": 11,
+			"trait": [
+			  {
+				"name": "Languages—",
```
fix language

luathebee commented on ArcanaOfTheAncientsBestiary.json:
```diff
@@ -8532,6 +8532,1595 @@
+			"speed": {
+			  "walk": 30,
+			  "burrow": 30
+			},
```
Traits are really broken lol

luathebee commented on ArcanaOfTheAncientsBestiary.json:
```diff
@@ -8532,6 +8532,1595 @@
+			  "restrained",
+			  "stunned"
+			],
+			"trait": [
```
fix languages and cr

luathebee commented on ArcanaOfTheAncientsBestiary.json:
```diff
@@ -8532,6 +8532,1595 @@
+			  "incapacitated",
+			  "restrained"
+			],
+			"fluff": {
```
lots of entries here have a \ every now and then. Should it be like this?

luathebee commented on ArcanaOfTheAncientsBestiary.json:
```diff
@@ -8532,6 +8532,1595 @@
+			"cha": 5,
+			"skill": {
+			  "anyonewithproficiency": "+1"
+			},
```
"anyonewithproficiency" is accpeted?

luathebee commented on ArcanaOfTheAncientsBestiary.json:
```diff
@@ -8532,6 +8532,1595 @@
+			],
+			"trait": [
+			  {
+				"name": "Languages—or one",
```
fix languages and cr

pcoffey42 committed:
Fixes for the beastiary json.

pcoffey42 commented:
I went through and fixed the issues and imported into the 5e.Tools homebrew tool and foundry for testing. I checked each of the creatures and fixed errors as I found them. Most were in the T-Z monsters, but I saw and fixed a few minor issues in the earlier entries as well, if you want to go over them:

-The Mlox had a duplicate "type" entry (making it an "item" instead of a "construct").
-There were a couple of visual bugs with languages (Having "Language" set to "" causes a blank Languages entry, but if you delete the field altogether it properly displays null languages as "-", which seems to be 5e convention.)

luathebee reviewed (approved):
Looks great! Thanks for fixing and going over previous entries as well :D
